### PR TITLE
Fix error when paper key owner not found

### PR DESCRIPTION
**Pull request by patrickxb**

### CODE REVIEW

File: go/engine/login_provision.go [hunk 1]
```diff
@@ -355,19 +359,25 @@ func (e *loginProvision) getValidPaperKey(ctx *Context) (*keypair, error) {
...
+			default:
 				lastErr = err
 			}
+
 			continue
 		}
 
 		if uid.NotEqual(e.arg.User.GetUID()) {
-			e.G().Log.Debug("paper key (%s) entered was for a different user", prefix)
-			lastErr = fmt.Errorf("paper key (%s) valid, but for %s, not %s", prefix, uid, e.arg.User.GetUID())
```
Comment (patrickxb) [inline go/engine/login_provision.go]: This is a horrible error to show to the user...

Comment (maxtaco) [reply go/engine/login_provision.go]: can this error still make it to the debug logs?

Comment (patrickxb) [reply go/engine/login_provision.go]: I thought it was bad to show it there too...but i can put it back.  Just seemed like if someone randomly entered someone else's paper key, we shouldn't advertise it.

Comment (maxtaco) [reply go/engine/login_provision.go]: oh you're right. ok, forget it.

**Pull request merged**